source/stylesheets/programmer.less

```diff
@@ -259,6 +259,7 @@ input:checked + .slider:before {
 #objects {
   margin: 0;
   overflow: scroll;
+  overflow-x: hidden;
```

Comment:
`overflow` is a shorthand property. `overflow: scroll` sets both `overflow-x` and `overflow-y`, and then we override `overflow-x`.

Better to set `overflow-x: hidden; overflow-y: scroll;` explicitly, don't you think?

Reply:
oops